docs/api/valueid.md

```diff
@@ -60,6 +61,7 @@ interface ValueMetadataAny {
 -   `description`: A description of the value
 -   `label`: A human-readable label for the property
 -   `ccSpecific`: CC specific information to help identify this value [(see below)](#CC-specific-fields)
+-   `setValueOptions`: Parameters that can be passed as `options` to the [`setValue`](api/node.md#setvalue) command for this value.
```

Comment:
Is this name correct?

Reply:
good catch, way to be on vacation. Will submit a new PR

Reply:
Whoops I forgot to change that when I renamed things